include/grpc/impl/codegen/log.h

```diff
@@ -63,6 +61,8 @@ GPRAPI void gpr_log(const char* file, int line, gpr_log_severity severity,
 
 GPRAPI int gpr_should_log(gpr_log_severity severity);
 
+GPRAPI int gpr_should_log_stacktrace(gpr_log_severity severity);
```

Comment:
do we really need C Core API for this? If it's test only why not keep it somewhere in src/

Reply:
I removed it from here so that it won't be part of API.